modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,58 @@
-
-* An Amphora load balancer VM is deployed per OpenShift Service with
-the default Octavia load balancer driver (Amphora driver). If the environment is
-resource constrained, creating a large amount of Services could be a problem.
-* Depending on the Octavia version, UDP listeners are not supported. This means
-that OpenShift UDP Services are not supported.
-* There is a known limitation of Octavia not supporting listeners on different
-protocols, like UDP and TCP, on the same port. Thus, Services exposing the same
-port for different protocols are not supported.
-* Due to the above UDP limitations of Octavia, Kuryr forces Pods to use TCP
-for DNS resolution. This is set with the `use-vc` option in `resolv.conf`. This
-might be a problem for Pods running Go applications compiled with the `CGO_DEBUG`
-flag disabled, as that uses the `go` resolver that only leverages UDP and is not
-considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a
-problem also for musl-based containers as its resolver does not support the
-`use-vc` option. This includes images built from `alpine`.
+There are a number of known limitations when using Kuryr SDN.
+
+[discrete]
+[id="openstack-resource-limitations_{context}"]
+== {rh-openstack} resource limitations
+
+* An Amphora load balancer VM is deployed per OpenShift Service that uses the
+default Octavia load balancer driver (Amphora driver). Creating too many Services
+can cause you to run out of resources.
+
+[discrete]
+[id="openstack-version-limitations_{context}"]
+== {rh-openstack} version limitations
+
+There are limitations when using Kuryr SDN that depend on the {rh-openstack}
+version.
+
+* Octavia {rh-openstack} versions before 16 do not support UDP listeners. Therefore,
+OpenShift UDP services are not supported.
+
+* Octavia {rh-openstack} versions before 16 cannot listen to multiple protocols on the
+same port. Services that expose the same port to different protocols, like TCP
+and UDP, are not supported.
+
+[IMPORTANT]
+====
+The OVN Octavia driver does not support listeners that use different protocols on
+any {rh-openstack} version.
+====
+
+[discrete]
+[id="openstack-go-limitations_{context}"]
+== {rh-openstack} environment limitations
+
+There are limitations when using Kuryr SDN that depend on your deployment environment.
+
+* Because of Octavia's lack of support for the UDP protocol, Kuryr forces Pods to use TCP
+for DNS resolution if:
+** The {rh-openstack} version is earlier than 16
+** The OVN Octavia driver is used
+
+On the host that runs the DHCP agent, the `use-vc` option in the file `resolve.conf`
+controls whether TCP is forced for DNS resolution.
+
+Go applications that are compiled without CGO support will use UDP only. In this case,
+the native Go resolver does not recognize the `use-vc` option in `resolve.conf`.
+
+[TIP]
+====
+CGO support is controlled by the `CGO_ENABLED` environment variable.
+CGO support is disabled by default on Go versions 1.12 and earlier.
```

Comment:
this is not true. What happens is that in 1.12 version or earlier, if you compile with CGO disabled, then only UDP is used. But, in 1.13, if UDP fails, TCP is tried, therefore fixing the issue